app/javascript/packs/src/settings/app.jsx

```diff
@@ -1,22 +1,40 @@
 import React from "react";
 import ReactDOM from "react-dom";
+import { BrowserRouter, Route, Switch, Redirect } from "react-router-dom";
```

Comment:
Right now we are not using History module. To reduce ESlint errors I did not include this module.